index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+

```

Comment:
- [ ] **OPTIONAL**
Please keep in mind that blank lines could lower the optimization of the website.
We only add them for good reasons such as the readability of the code when separating two logical pieces of code or two large sections apart.

> Please note that this is a mandatory requirement, and I'm sorry that it didn't show up for you in the earlier review.
You can see it and similar important notes right [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

_If you decided to change that, please apply it to similar situations in your code files._